rwgame/RWGame.cpp

```diff
@@ -797,4 +968,10 @@ void RWGame::globalKeyEvent(const SDL_Event& event)
 		break;
 	default: break;
 	}
+
+	std::string keyName = SDL_GetKeyName(event.key.keysym.sym);
```

Comment:
Did you see https://wiki.libsdl.org/SDL_TextInputEvent? The cheats all use latin chars so it's fine as is.